docs/concepts/vscode-extension.md

```diff
@@ -52,93 +50,47 @@ To develop in a fully configured remote development environment, follow the inst
+
+## Setup integrations and scripts environment
 
 Each integration or script in `XSOAR` runs on a different environnement, and has different dependencies.
-With the VSCode extension, you are able to open a workspace of the integration or a script with it's dedicated virtual environment.
+This feature will configure the integration or script, and will allow you to debug it easily and run unit tests.
+
+In addition, you will be able to open the integration environment in a new workspace with a virtual environment, for autocompletion.
 
 ### Usage
 
 - Go to the integration or a script.
-- Right-click it, and select **Open integration/script in virtual environnement**
-
-### Features
-
-When using it, the following actions are automatically performed:
-
-- Initializes a Python virtual environment based on the integration Docker image with all dependencies and test dependencies installed.
-- Copies the dependent files to the directory (`CommonServerPython.py` , `demistomock.py` and ApiModules if necessary).
-- Opens a new VSCode workspace that is configured with **pytest** and linting with **mypy** and **flake8** enabled. Run the tests with the `Test Explorer`. See the linting errors in the `Problems` tab.
-- Switches to the newly created virtual environment as the selected Python interpreter in this workspace.
-- Creates a configured `launch.json` in the workspace, to debug the integration easily with demistomock.
-- If the virtual environment already exists, a popup will be triggered to ask you whether to open the existing environment or create a new one.
+- Right-click it, and select **Setup integration/script environment**
+- There will be a popup asking you if you want to use the current workspace or open a new one with a virtual environment.
+  - Using the current workspace is quicker, but you may will not have autocompletion for some integrations.
+  - Opening a new workspace will take longer, but you will have autocompletion for all integrations.
+- With both options, you will be able to debug your integration/script easily, run unit tests, and see problems in the IDE.
 
 ## Debugging
 
-- First, [Open integrations and scripts in the Python virtual environment](#open-integrations-and-scripts-in-python-virtual-environment)
+- First, [Setup integrations and scripts environment](#setup-integrations-and-scripts-environment)
 - Then, read the [Debugging using your IDE](../integrations/debugging#Debugging%20using%20your%20IDE) section.
+- Go to the Run and Debug (⇧⌘D), and make sure that `Docker: debug (<integration>)` is selected.
+- Click on the green arrow or F5 to start debugging.
+
+## Run and Debug tests
 
-You can use Run and Debug (⇧⌘D) in VSCode (or click F5 on the Python file).
+- First, [Setup integrations and scripts environment](#setup-integrations-and-scripts-environment)
+- Go to the Run and Debug (⇧⌘D), and make sure that `Docker: debug tests (<integration>)` is selected.
```

Comment:
What is this?  (⇧⌘D)

Reply:
This is a shorcut for it (Command+Shift+D)
I will add a link to the offcial docs for it:
https://code.visualstudio.com/docs/editor/debugging#_run-and-debug-view